### ISSUE TITLE
Should I make pull requests for my CMake branches?

### ISSUE THREAD

**Issue by JeffreyO** [CelesteBlue-dev/PSVita-RE-tools#36]
Or just keep them on my own fork? Do you prefer for the project to stay only with Makefiles?

**Comment by CelesteBlue-dev** [comment]
Hello. Maybe it would be convenient you to make a pull request that adds a CMakefile into tool_name/src/cmakefile/.

Makefiles are in my opinion good to keep build working, without caring about CMake versions that could break compilation. You seem to use CMake because android aarch64 does not support Makefiles?

**Comment by JeffreyO** [comment]
Yeah. I use CMake because the setup and compilation on Termux is a lot easier. It adds all the flags needed for the platform and only needs minor modifications to get stuff working sometimes.

**Comment by JeffreyO** [comment]
I'm not sure if moving the CMakeLists.txt into a directory without the source code will work, I'll need to look into that.

Though, CMake shouldn't overwrite the original Makefile since you have to build out of source or the tool will complain and fail before generating the build-script.

**Comment by JeffreyO** [comment]
It looks like it's not widely recommended to place CMakeLists.txt into a subdirectory. Especially since it could cause some issues on more complex nested projects.
